### PR TITLE
Try to fix ci

### PR DESCRIPTION
**Pull request by KochetovNicolai**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: src/Core/Settings.h [hunk 1]
```diff
@@ -399,6 +399,7 @@ class IColumn;
     \
     /** Obsolete settings that do nothing but left for compatibility reasons. Remove each one after half a year of obsolescence. */ \
     \
+    M(UInt64, max_memory_usage_for_all_queries, 0, "Obsolete. Will be removed after 2020-10-20", 0) \
```
Comment (alexey-milovidov) [inline src/Core/Settings.h:402]: Clarification needed.

Comment (KochetovNicolai) [reply src/Core/Settings.h:402]: This setting is used in tests

Comment (alexey-milovidov) [reply src/Core/Settings.h:402]: All these tests are incorrect and should be changed accordingly.

Comment (KochetovNicolai) [reply src/Core/Settings.h:402]: Only tests from Arcadia depend on this setting now. Our colleagues will fix this tests, and then we remove this setting.

**Pull request merged**